gbpservice/nfp/orchestrator/drivers/orchestration_driver.py

```diff
@@ -942,6 +942,9 @@ def plug_network_function_device_interfaces(self, device_data,
 
         token = device_data['token']
         tenant_id = device_data['tenant_id']
+        image_name = self._get_image_name(device_data)
```

Comment:
suresh this is not needed, usage of this API is optimized, update vendor data is called already in the path. Check once. If this is the first place then remove it in the path where it is called after this.